
public class exceptionsample {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
		
		try
		{
		String abc="adadsd";
		int a=Integer.parseInt(abc);
		
		System.out.println(abc);
		}
		catch(ArithmeticException e)
		{
			
		}

	}

}
